assets/scss/pages/page.scss

```diff
@@ -0,0 +1,34 @@
+.p-page__container {
+    padding: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+
+    @include bp(medium) {
+        padding: 3rem 5rem;
+        gap: 3rem;
+    }
```

Comment:
@mgbelegu wouldn't it be better if we make a general container component for these?

Reply:
@marcinkrzeminski what do you mean by this? it's better to create a "div" element for example with class name .c-container that handles this styling?

Reply:
@mgbelegu yes exactly. Move out outside of the page.scss and make a separate component out of it.